spec/models/run_spec.rb

```diff
@@ -89,6 +89,24 @@
     end
   end
 
+  context 'with bests in all segments' do
+    let(:run) { FactoryBot.create(:with_segments_bests_run) }
+
+    it 'has a realtime sum of best in ms' do
+      run.parse_into_db
+      run.reload
+      expect(run.realtime_sum_of_best_ms).to be_within(run.segments.count)
```

Comment:
I had to use `be_within` here because rounding is done differently when storing the sum_of_best counts and when storing each individual segment. Rounding is actually done by truncating in both cases, but for sum_of_best we first sum all times and then truncate, and for segments we truncate each time individually.

In the sample file, there are times like `132.740999`. This means that the corresponding segment gets saved as `132740`, but summing up a couple of those makes the sum_of_best result a couple of seconds longer. I wonder if it would make sense to round instead of truncate when saving the segments (`132.740999` would be `132741` instead of `132740`) but that would be a way bigger change than what I intended to do so I assume there can be at most a difference of milliseconds equal to the number of segments in the worst case.